test/testdata/goldens/dependency_services/dependency_services_test/multibreaking.txt

```diff
@@ -0,0 +1,156 @@
+packages:
+  bar:
+    dependency: "direct main"
+    description:
+      name: bar
+      url: "http://localhost:$PORT"
+    source: hosted
+    version: "1.0.0"
+  foo:
+    dependency: "direct main"
+    description:
+      name: foo
+      url: "http://localhost:$PORT"
+    source: hosted
+    version: "1.0.0"
+sdks:
+  dart: ">=0.1.2 <1.0.0"
+-------------------------------- END OF OUTPUT ---------------------------------
+
+## Section list
+$ dependency_services list
+{
+  "dependencies": [
+    {
+      "name": "bar",
+      "version": "1.0.0",
+      "kind": "direct",
+      "constraint": "^1.0.0"
+    },
+    {
+      "name": "foo",
+      "version": "1.0.0",
+      "kind": "direct",
+      "constraint": "^1.0.0"
+    }
+  ]
+}
+
+-------------------------------- END OF OUTPUT ---------------------------------
+
+## Section report
+$ dependency_services report
+{
+  "dependencies": [
+    {
+      "name": "bar",
+      "version": "1.0.0",
+      "kind": "direct",
+      "latest": "2.0.0",
+      "constraint": "^1.0.0",
+      "compatible": [],
+      "singleBreaking": [],
+      "multiBreaking": [
+        {
+          "name": "bar",
+          "version": "2.0.0",
+          "kind": "direct",
+          "constraint": "^2.0.0",
+          "previousVersion": "1.0.0",
+          "previousConstraint": "^1.0.0"
+        },
+        {
+          "name": "foo",
+          "version": "3.0.1",
+          "kind": "direct",
+          "constraint": "^3.0.1",
+          "previousVersion": "1.0.0",
+          "previousConstraint": "^1.0.0"
+        }
+      ]
+    },
+    {
+      "name": "foo",
+      "version": "1.0.0",
+      "kind": "direct",
+      "latest": "3.0.1",
+      "constraint": "^1.0.0",
+      "compatible": [
+        {
+          "name": "foo",
+          "version": "1.5.0",
+          "kind": "direct",
+          "constraint": "^1.0.0",
+          "previousVersion": "1.0.0",
+          "previousConstraint": "^1.0.0"
+        }
+      ],
+      "singleBreaking": [
+        {
+          "name": "foo",
+          "version": "2.0.0",
+          "kind": "direct",
+          "constraint": "^2.0.0",
+          "previousVersion": "1.0.0",
+          "previousConstraint": "^1.0.0"
+        }
+      ],
+      "multiBreaking": [
+        {
+          "name": "foo",
+          "version": "3.0.1",
+          "kind": "direct",
+          "constraint": "^3.0.1",
+          "previousVersion": "1.0.0",
+          "previousConstraint": "^1.0.0"
+        },
+        {
+          "name": "bar",
+          "version": "2.0.0",
+          "kind": "direct",
+          "constraint": "^2.0.0",
+          "previousVersion": "1.0.0",
+          "previousConstraint": "^1.0.0"
+        }
+      ]
+    }
+  ]
+}
+
+-------------------------------- END OF OUTPUT ---------------------------------
+
+## Section apply
+$ echo '{"dependencyChanges":[{"name":"foo","version":"3.0.1","constraint":"^3.0.0"},{"name":"bar","version":"2.0.0"}]}' | dependency_services apply
```

Comment:
Shouldn't there also be a `constraint` for `bar`? Just asking... I'm not actually sure we require that... probably because dependabot loses this information...

Reply:
I don't think so. This is a single-breaking change. Only foo should have its constraint updated.

Also depandabot would lose this information

Reply:
I think this is the multibreaking example.... but yes dependabot will probably loose this information.

Reply:
Ah you're right.